src/util/inject.js

```diff
@@ -64,10 +64,12 @@ export const inject = (asyncFunc, args = []) => new Promise((resolve, reject) =>
   callbacks.set(callbackId, [resolve, reject]);
 
   const script = document.createElement('script');
+  const name = `xkit$${asyncFunc.name || 'injected'};
```

Comment:
I LOVE TYPOS I LOVE TYPOS I LOVE TYPOS
```suggestion
  const name = `xkit$${asyncFunc.name || 'injected'}`;
```

Reply:
I blame VS Code for automatically creating closing quotes/brackets/parens when you type an opening one. I've gotten lazy.